release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -116,6 +116,11 @@ With this release, there are several updates to the *Developer* perspective of t
 ** Cluster roles accessible to a project
 ** Actions visible on the *+Add* page
 ** Item types in the *Developer Catalog*
+* See the common updates to the *Pipeline details* and *PipelineRun details* page visualization performing the following actions:
+** Use the mouse wheel to change the zoom factor.
+** Hover over the tasks and see the task details.
+** Use the zoom in, zoom out, fit to screen, and reset view features using the standard icons.
```

Comment:
Use the standard icons to zoom in, zoom out, fit to screen and reset the view.

Reply:
"... fit to screen, and reset the view." 

Serial comma!

Reply:
Based on comments for Line 122, I will rephrase it as follows:
"Use the standard icons to zoom in, zoom out, fit to screen, and reset the view."